### ISSUE TITLE
Path param not showing in Path params editors when using multiple env variable

### ISSUE THREAD

**Issue by jbuiquan** [usebruno/bruno#3067]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

When I use two environment variable in my url, the path param is not added to the path param editor and i can't set a value.
When I use only one environment variable it works again.
I don't know why the path param gets stuck in red too.

I'm using the .deb package.

### .bru file to reproduce the bug

```
meta {
  name: Test
  type: http
  seq: 2
}

get {
  url: {{api_base_url}}:{{api_port}}/privileges/:id
  body: none
  auth: none
}
```

### Screenshots/Live demo link

With 2 variables
![image](https://github.com/user-attachments/assets/d57efd7d-d4db-48f0-906f-ac047581c2ca)

With 1 variable
![image](https://github.com/user-attachments/assets/ac722c32-6bf9-466b-9bc4-9430a32c9621)

**Comment by apoorvyadav1111** [comment]
Hey, If you are not picking this up, I would like to work on this issue, Please assign this to me.

**Comment by apoorvyadav1111** [comment]
Hi @sanjai0py, I have raised a PR #3107 to address this issue. I would appreciate your feedback.

**Comment by helloanoop** [comment]
This will be available in upcoming `v1.29.0` release

**Comment by abhishekkmr13** [comment]
I am still facing this issue on v1.40.0

My sample endpoint is:
{{host}}/user//{{user_id}}

I am getting below error:
405 Method Not Allowed


When I try inspecting the curl for this call then I don't find user_id correctly mapped to it.
curl --request GET \
  --url https://abc.com/user/ \
  --header 'accept: application/json'
